src/smtp_util.erl

```diff
@@ -88,11 +88,9 @@ compute_cram_digest(Key, Data) ->
     Bin = hmac_md5(Key, Data),
     list_to_binary([io_lib:format("~2.16.0b", [X]) || <<X>> <= Bin]).
 
--ifdef(OTP_RELEASE).
```

Comment:
I'm not sure about this construct. But it seems to me it could be fine this way also.
Take with caution. I do not speak Erlang.